Automattic/course
# Design navbar as per design for desktop and mobile

Implements https://github.com/Automattic/sensei-theme/issues/7

### Proposed Changes

- Added navbar fonts
- Added custom GB style for navbar URL to make it look like a button
- Made navbar menu items gaps match the design, on mobile and desktop
- Made navbar spacings match the design on mobile and desktop
- Removed site description from header
- Changes font size, line height, link decoration for site title block, for both desktop and mobile
- Right colors in background, text, icons for navbar URL, navbar button, navbar URL mobile, navbar button mobile
- Separate font size for Normal URLs and Button style URLs in navbar
- Make header sticky, add class to header when sticking on top using scroll event

### Testing instructions
- Go to header template from site editor, click on navigation block
- Add the menu items according to the design
- Select the last item in the navbar URLs, on the right, click on Button
- Now save and go to frontend
- Make sure all the changes listed above work and the design looks as expected

https://user-images.githubusercontent.com/6820724/194651845-b58aca2b-88f2-4f7b-8466-47c933caefa6.mov

### Screenshots
Desktop navbar
<img width="1370" alt="Screen Shot 2022-10-06 at 10 03 40 AM" src="https://user-images.githubusercontent.com/6820724/194213548-724f0626-2431-4760-a970-f8272070f19e.png">

Desktop sticky
<img width="1370" alt="Screen Shot 2022-10-06 at 10 21 06 AM" src="https://user-images.githubusercontent.com/6820724/194213613-f835925c-6afb-4036-a97e-516f416ba8e5.png">

Mobile navbar
<img width="524" alt="Screen Shot 2022-10-06 at 10 03 09 AM" src="https://user-images.githubusercontent.com/6820724/194213693-8c9fc3a9-e45a-4cfc-9891-fb1cf239127b.png">
